Restructure (#542)
hacs/documentation: source/docs/use/manage/type/python_script.md

title: Python Script
---

Python Scripts is a part of the Home Assistant [`python_script` integration.](https://www.home-assistant.io/integrations/python_script/)

## Enable Python Scripts in Home Assistant

Python Scripts are automagically enabled in HACS if you have them enabled in Home Assistant.

> To enable [Python Scripts](https://www.home-assistant.io/integrations/python_script/) in Home Assistant, add this to your [`configuration.yaml` file](https://www.home-assistant.io/docs/configuration/)

Author: ```suggestion
To enable [Python Scripts](https://www.home-assistant.io/integrations/python_script/) in Home Assistant, add the following line to your [`configuration.yaml` file](https://www.home-assistant.io/docs/configuration/)

```

title: Python Script
---

> Python Scripts is a part of the Home Assistant [`python_script` integration.](https://www.home-assistant.io/integrations/python_script/)

Author: ```suggestion
Python Scripts is a part of the Home Assistant [`python_script` integration](https://www.home-assistant.io/integrations/python_script/).

```

To enable [Python Scripts](https://www.home-assistant.io/integrations/python_script/) in Home Assistant, add the following line to your [`configuration.yaml` file](https://www.home-assistant.io/docs/configuration/)

> 

Author: ```suggestion

```
